oat-sa/extension-tao-delivery-rdf
# Fix/TAO-10313/fix export compiled delivery

__Related to task:__ https://oat-sa.atlassian.net/browse/TAO-10313
__Description:__ Fix exportCompiledDelivery

__How to test:__ 
1. create delivery
2. run the command in the console "php index.php "\oat\taoDeliveryRdf\model\export\ExportAssemblyClass" http://www.tao.lu/Ontologies/mpart.rdf#Form data/exports/"

__Expected Result:__ In the "data/exports/" folder, an archive appears that contains delivery
